src/community/taskmanager/src/main/java/org/geoserver/taskmanager/fileservice/FileService.java

```diff
@@ -25,7 +25,7 @@
      * 
      * @return description
      */
-    public String getDescription();
+        String getDescription();
 
```

Comment:
Deze snap ik niet?

Reply:
Intellij klaagt altijd over public in interfaces :) Modifier 'public' is redundant for interface methods

Reply:
Ah ja dat is nen interface, juist. Dan heb ik het er ook liever niet in.